.bazelci/presubmit.yml

```diff
@@ -183,14 +183,14 @@ tasks:
     test_flags:
       - --test_output=errors
   testing_java_image:
-    platform: ubuntu2104
+    platform: ubuntu2004
```

Comment:
Any specific reason for the downgrade? is one of the deps we are pulling in limiting us to this? Might be worth documenting.

Reply:
Documented in my PR description. Copied here for visibility:
> Using ubuntu2004 because the latest ubuntu2204 does not have docker installed.

I also created a https://github.com/bazelbuild/continuous-integration/pull/1487 so we can eventually upgrade to 2204